contracts/Delay.sol

```diff
@@ -115,7 +121,7 @@ contract Delay is AragonApp, IForwarder {
     */
     function execute(uint256 _delayedScriptId) external {
         require(canExecute(_delayedScriptId), ERROR_CAN_NOT_EXECUTE);
-        runScript(delayedScripts[_delayedScriptId].evmCallScript, new bytes(0), new address[](0));
+        runScript(delayedScripts[_delayedScriptId].evmCallScript, new bytes(0), scriptRunnerBlacklist);
```

Comment:
Looks good! This definitely disallows direct reentrancy, but I would recommend also doing `delete delayedScripts[_delayedScriptId]` before calling `runScript`. As it stands now, this can still reenter and run the same script twice, just not by directly calling `Delay` again.

Reply:
I've decided to remove the blacklisting of the Delay App's address for the reason @fabriziovigevani raised. We may want to delay changing some of the Delay App's configuration.

However, I have moved the `delete delayedScripts[_delayedScriptId]` to before the `runScript()` call.

Reply:
He raised a good point. I think this is the most reasonable approach, and there's always `pauseExecution` and `cancelExecution` in case `DelayedScripts` appear dangerous.